src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  box-sizing: border-box;
```

Comment:
move to `*` selector to prevent code duplicating

Reply:
![image](https://user-images.githubusercontent.com/108959592/207295004-b5504041-1649-4cc8-b436-ee6dc0596d23.png)
the gap property doesn`t set

Reply:
gap properpy works, sorry. i just had to set it in .calendar class